File: riesi.cs
/* dotnet script riesi.cs */
bool   reisi = false;
string input;

Console.WriteLine($"Reisi: {reisi.ToString()}");
Console.Write("Reisi?");

input = Console.ReadLine().ToLower();
reisi = (input == "y");

Console.WriteLine($"Reisi: {reisi.ToString()}");

File: riesi.vb
Dim riesi As Boolean
Dim input As String

Console.WriteLine($"Riesi: {riesi.ToString()}")
Console.Write("Riesi?")

input = Console.ReadLine().ToLower()
riesi = (input = "y")

Console.WriteLine($"Riesi: {riesi.ToString()}")
